sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -236,6 +237,23 @@ def _test_pipeline_workflow(self,
         compiled = list(yaml.safe_load_all(f))
       if GENERATE_GOLDEN_YAML:
         with open(golden_yaml_file, 'w') as f:
+          f.write(textwrap.dedent("""\
```

Comment:
can you move this text into a variable at the top of the file

Reply:
done